lennoxs30api/s30api_async.py

```diff
@@ -930,6 +932,20 @@ def processEquipments(self, message) -> bool:
                             self.single_setpoint_mode = False
                         self._dirty = True
                         self._dirtyList.append("single_setpoint_mode")
+                for diagnostic in equipment.get("equipment", {}).get("diagnostics", []):
+                    # look up by name instead of id
+                    # different equipments reuse IDs for different points
+                    # and this diagnostic view is less likely to be translated
+                    diagnostic_name = diagnostic.get("diagnostic", {}).get("name")
+                    diagnostic_value = diagnostic.get("diagnostic", {}).get("value")
+                    if diagnostic_name == "Inverter Input Voltage":
+                        self.diagInverterInputVoltage = float(diagnostic_value)
```

Comment:
What happens if the conversion fails?

Reply:
To keep the attr_updater simple, I'll skip the conversion here and leave it to the HA sensor